src/config.rs

```diff
@@ -39,21 +39,27 @@ pub struct Config {
     pub overwrite: bool,
 }
 
+impl std::default::Default for Config {
```

Comment:
Can we use `std::default::Default` on top and use only `Default` here?

Reply:
Yeah for sure!

Reply:
Actually on second thought we don't even need to bring in the import. It's brought in as a part of `core` for edition 2021.

Reply:
(Which is why all of the other references to `Default` work even though I didn't scope them 🙃 )

Reply:
Oh nice haha